### ISSUE TITLE
NullPointerException error caused by ChatSerializer.fromRaw 

### ISSUE THREAD

**Issue by lgou2w** [lgou2w/ldk#93]
Caused:
```kotlin
val raw = "§lHello" // not a color, is formatter
ChatSerializer.fromRaw(raw) // throw NullPointerException
```
See:
https://github.com/lgou2w/ldk/blob/16f52026937e1fc17e66f8a679ce0219e1442e25/ldk-chat/src/main/kotlin/com/lgou2w/ldk/chat/ChatSerializer.kt#L189-L193

**Comment by lgou2w** [comment]
Will not be released in the hot fix version, please use the latest snapshot :(

**Comment by lgou2w** [comment]
Fixed in `0.1.9-SNAPSHOT`.
